test/attr/attr_dynamic_infer.swift

```diff
@@ -37,17 +37,17 @@ extension Sub {
 
   // CHECK: @objc override dynamic var prop: Super
   override var prop: Super {
-    // CHECK: @objc dynamic get
+    // CHECK: @objc get
     get { return Super() }
-    // CHECK: @objc dynamic set
+    // CHECK: @objc set
     set { }
   }
 
   // CHECK: @objc override dynamic subscript(sup: Super) -> Super
   override subscript(sup: Super) -> Super {
-    // CHECK: @objc dynamic get
+    // CHECK: @objc get
     get { return sup }
-    // CHECK: @objc dynamic set
```

Comment:
I hate to say it, but it really might be worth writing a SILGen test for these (just that a call to them has the dynamic bit set). I could totally see us regressing here otherwise.

Reply:
This can be a follow-up commit though.